app/modules/util/config/index.ts

```diff
@@ -0,0 +1,6 @@
+import { env } from 'process';
```

Comment:
You tend to love OOP a lot :D 

A simpler way could have been - 

```
const DB_HOST: string = env.DB_HOST || '';
const PORT: string = env.PORT || '3000';

export { DB_HOST, PORT };
```

Any importer, can just select what env variable they need - 

```
import { PORT } from "./config";
```